www/configurator.html

```diff
@@ -132,7 +132,9 @@
 
 
         </div>
-
+        <button type="button" class="btn btn-primary mr-3" @click="downloadLogs()" id="logs"
```

Comment:
Attach screenshot to PR, please. It is more comfortable to estimate designed solution.

Reply:
![image](https://user-images.githubusercontent.com/20268352/61180215-fbc79680-a61b-11e9-8c68-c3dccd42f705.png)